Net/SMTP.php

```diff
@@ -912,12 +912,10 @@ protected function authGSSAPI($uid, $pwd, $authz = '')
             return PEAR::raiseError('No Kerberos service principal set', 2);
         }
 
-        if (!$this->gssapi_cname) {
-            return PEAR::raiseError('No Kerberos credentials cache set', 2);
+        if ($this->gssapi_cname) {
```

Comment:
Should this be a `!empty()` test, or is it valid to set `KRB5CCNAME` to an empty string?